### ISSUE TITLE
Chainable methods enumeration

### ISSUE THREAD

**Issue by jmm** [tgriesser/knex#1536]
[Chainable Methods](http://knexjs.org/#Chainable) says:

> The following three methods [...]

But there are more than 3. Probably should just say:

> The following methods [...]

**Comment by avinoth** [comment]
I can fix this along with #1535. Anybody working on the docs already?

**Comment by rhys-vdw** [comment]
@jmm thanks for the heads up. if you find issues like this, just open a PR. The docs are regularly tweaked via community contributions.

**Comment by johanneslumpe** [comment]
@rhys-vdw hijacking, but what about populating docs almost purely from doc blocks? That could help a lot because the code itself would be documented and we could just turn those into a documentation of sorts.

**Comment by rhys-vdw** [comment]
@johanneslumpe yeah, totally. There's a bit of a discussion in #1125 if you'd like to throw some ideas out or are interested in helping.

**Comment by jmm** [comment]
@rhys-vdw Thanks! Totally appreciate where you're coming from. However, I don't always have the opportunity to do a PR when I find a problem, and have been having some bad experiences with doc PRs (not here) that have been demotivating me from doing them in general. So, would you prefer to have an issue logged rather than nothing if those are the options on my end? That way it's at least surfaced, and others like @avinoth and @johanneslumpe may have the opportunity to take action. Or I might end up being able to eventually do a PR and can remember the problem because of the GH issue. Let me know what you think -- I don't want to make your work harder. Thanks!!

**Comment by mercmobily** [comment]
The good news is that this issue has been resolved through the natural evolution of our documentation. The current documentation at https://knexjs.org/guide/query-builder.html no longer contains the problematic text "The following three methods" that you originally reported. The documentation has been significantly restructured and improved since 2016, and this specific wording issue no longer exists.

We've had ongoing efforts to improve documentation quality, as evidenced by the many documentation-related issues and contributions over the years. The community has been actively working on making our docs more accurate and comprehensive.

This issue can be safely closed as the original problem has been resolved through documentation updates.
